static/app/views/dashboards/widgets/timeSeriesWidget/plottables/area.tsx

```diff
@@ -0,0 +1,74 @@
+import type {LineSeriesOption} from 'echarts';
+
+import LineSeries from 'sentry/components/charts/series/lineSeries';
+
+import type {Plottable, TimeSeries} from '../../common/types';
+import {splitSeriesIntoCompleteAndIncomplete} from '../splitSeriesIntoCompleteAndIncomplete';
+import {timeSeriesItemToEChartsDataPoint} from '../timeSeriesItemToEChartsDataPoint';
+
+interface AreaOptions {
+  color?: string;
+  dataCompletenessDelay?: number;
+}
+
+export class Area implements Plottable {
```

Comment:
Plottable sounds like a good candidate for an abstract class

Reply:
Great suggestion, thanks! Done